workflow_tests/finishReleaseCycle.test.ts

```diff
@@ -1,12 +1,14 @@
-const path = require('path');
-const kieMockGithub = require('@kie/mock-github');
-const utils = require('./utils/utils');
-const assertions = require('./assertions/finishReleaseCycleAssertions');
-const mocks = require('./mocks/finishReleaseCycleMocks');
-const ExtendedAct = require('./utils/ExtendedAct').default;
+import {MockGithub} from '@kie/mock-github';
+import path from 'path';
+import assertions from './assertions/finishReleaseCycleAssertions';
+import * as mocks from './mocks/finishReleaseCycleMocks';
```

Comment:
```suggestion
import mocks from './mocks/finishReleaseCycleMocks';
```

Merge with `main` first